composer.json

```diff
@@ -10,52 +10,57 @@
         }
     ],
     "autoload": {
-        "psr-0": { "": "src/" }
+        "psr-4": { "": "src/" },
+        "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
     },
+    "autoload-dev": {
+        "classmap": [ "app/tests/WebTestCase.php" ]
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/WouterJ/SonataAdminBundle.git"
+        }
+    ],
     "require": {
-        "php": "^5.3.9|^7.0",
-        "symfony/symfony": "^2.3,<2.8.1|>=2.8.2",
-        "symfony/assetic-bundle": "^2.3",
+        "php": "^5.5.9|^7.0",
+        "symfony/symfony": "^3.0",
+        "symfony/assetic-bundle": "^2.8",
         "symfony/swiftmailer-bundle": "^2.3",
-        "symfony/monolog-bundle": "^2.3",
-        "twig/extensions": "^1.0",
-        "sensio/distribution-bundle": "^3.0",
-        "sensio/framework-extra-bundle": "^2.3",
+        "symfony/monolog-bundle": "^2.8",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
 
-        "symfony-cmf/symfony-cmf": "1.3.*",
-        "symfony-cmf/simple-cms-bundle": "1.3.*",
-        "symfony-cmf/search-bundle": "1.2.*",
-        "symfony-cmf/seo-bundle": "1.2.*",
-        "symfony-cmf/routing-auto-bundle": "1.1.*",
+        "symfony-cmf/symfony-cmf": "2.0.*",
+        "symfony-cmf/seo-bundle": "2.0.*",
+        "symfony-cmf/routing-auto-bundle": "2.0.*",
 
         "jackalope/jackalope-doctrine-dbal": "1.2.*",
         "jackalope/jackalope-jackrabbit": "1.2.*",
-        "doctrine/doctrine-bundle": "^1.2",
+        "doctrine/doctrine-bundle": "^1.6",
         "doctrine/data-fixtures": "^1.0",
         "doctrine/doctrine-cache-bundle": "^1.0",
         "doctrine/phpcr-odm": "^1.3",
 
         "sonata-project/cache-bundle": "^2.1",
-        "sonata-project/doctrine-phpcr-admin-bundle": "1.2.*",
-        "sonata-project/admin-bundle": "^2.3.3",
-        "sonata-project/translation-bundle": "1.0.0|^1.0.2",
+        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
+        "sonata-project/admin-bundle": "dev-fix-bug-admin-groups as 3.1.0",
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/pull/3903 has been merged.

not sure why this fails in install, you do tell composer to look at your fork of the admin-bundle repo...